en/02_Developer_Guides/03_Forms/How_Tos/05_Simple_Contact_Form.md

```diff
@@ -135,12 +135,11 @@ class ContactPageController extends PageController
 }
 ```
 
-[hint]
-Caution: This form is prone to abuse by spammers,
-since it doesn't enforce rate limitation and doesn't implement any checks for bots.
-We recommend to use a validation service like the ["recaptcha" module](https://github.com/silverstripe/silverstripe-spamprotection)
-for better security.
-[/hint]
+> [!CAUTION]
+> Caution: This form is prone to abuse by spammers,
+> since it doesn't enforce rate limitation and doesn't implement any checks for bots.
+> We recommend to use a validation service like the ["recaptcha" module](https://github.com/silverstripe/silverstripe-spamprotection)
+> for better security.
```

Comment:
Intentionally changed to `CAUTION` because that's clearly more appropriate for this content